linkml-schema/kbmodel.yaml

```diff
@@ -96,34 +80,463 @@ classes:
+
+  tissue dissecting:
+    description: >-
+      A process that takes a brain section and produces a tissue sample.
+    is_a: activity
+    mixins: 
+      - prov activity
+    slot_usage:
+      used:
+        range: brain section
+      generated:
+        range: tissue sample
+
+  cell dissociation:
+    description: >-
+      A process that takes a tissue sample and produces a dissociated cell sample.
+    is_a: activity
+    mixins: 
+      - prov activity
+    slot_usage:
+      used:
+        range: tissue sample
+      generated:
+        range: dissociated cell sample
+
+  cell enrichment:
+    description: >-
+      A process that takes a dissociated cell sample and produces an enriched cell sample.
+    is_a: activity
+    mixins: 
+      - prov activity
+    slot_usage:
+      used:
+        range: dissociated cell sample
+      generated:
+        range: enriched cell sample 
+
+  cell barcoding:
+    description: >-
+      A process that takes an enriched cell sample and produces a barcoded cell sample.
+    is_a: activity
+    mixins: 
+      - prov activity
+    slot_usage:
+      used:
+        range: enriched cell sample
+      generated:
+        range: barcoded cell sample
+
+  cdna amplification:
+    description: >-
+      A process that takes a barcoded cell sample and produces an amplified cDNA sample.
+    is_a: activity
+    mixins: 
+      - prov activity
+    slot_usage:
+      used:
+        range: barcoded cell sample
+      generated:
+        range: amplified cdna
+  
+  library construction:
+    description: >-
+      A process that takes an amplified cDNA sample and produces a library.
+    is_a: activity
+    mixins: 
+      - prov activity
+    slot_usage:
+      used:
+        range: amplified cdna
+      generated:
+        range: library
+  
+  library aliquoting:
+    description: >-
+      A process that takes a library and produces an library aliquot.
+    is_a: activity
+    mixins: 
+      - prov activity
+    slot_usage:
+      used:
+        range: library
+      generated:
+        range: library aliquot
+
+  library pooling:
+    description: >-
+      A process that takes a library aliquot and produces a library pool.
+    is_a: activity
+    mixins: 
+      - prov activity
+    slot_usage:
+      used:
+        range: library aliquot
+      generated:
+        range: library pool
+
+
+  prov entity:
+    description: >-
+      Based off prov:Entity; an entity is a physical, digital, conceptual, or other kind of thing with some fixed aspects; entities may be real or imaginary.
+    slots:
+      - wasDerivedFrom
+      - wasGeneratedBy
+
+
+  # 028 - donors.ipynb
+  donor:
+    is_a: entity
+    mixins:
+      - thing with taxon
+      - prov entity
+    description: >-
+      A person or organism that is the source of a biological sample.
+    slots:
+      - sex
+      - date of birth
+      - date of death
+      - age at death
+      - full genotype
+
+
+
+  # 027 - brain_extraction.ipynb
```

Comment:
we should consider what a brain is. it seems that it is a type of biosample that is attributed to a donor.